lib/internal/crypto/random.js

```diff
@@ -119,6 +120,55 @@ function randomFill(buf, offset, size, cb) {
   _randomBytes(buf, offset, size, wrap);
 }
 
+// Largest integer we can read from a buffer.
+// e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
+const RAND_MAX = 281474976710655;
+
+function randomInt(min, max, cb) {
+  // randomInt(max, cb)
+  if (typeof max === 'function') {
+    cb = max;
+    max = min;
+    min = 0;
+  }
+  if (typeof cb !== 'function') {
+    throw new ERR_INVALID_CALLBACK(cb);
+  }
+  if (!NumberIsInteger(min)) {
+    throw new ERR_INVALID_ARG_TYPE('min', 'integer', min);
+  }
+  if (!NumberIsInteger(max)) {
+    throw new ERR_INVALID_ARG_TYPE('max', 'integer', max);
+  }
+  if (!(max > min)) {
+    throw new ERR_OUT_OF_RANGE('max', `> ${min}`, max);
+  }
+  if (!(max <= RAND_MAX)) {
+    throw new ERR_OUT_OF_RANGE('max', `<= ${RAND_MAX}`, max);
+  }
```

Comment:
Should this check be for `max - min`, i.e. `range`, instead of just `max`? (because negative `min` would make `range` larger than `max`)

Reply:
Good catch 👍  fixed